presto-docs/src/main/sphinx/release/release-0.261.rst

```diff
@@ -0,0 +1,50 @@
+=============
+Release 0.261
+=============
+
+**Highlights**
+==============
+* Improved performance of sum and average aggregations on decimals.
+* Support for LocationProvider supplied paths in Iceberg connector.
+* Column comments now available in JDBC connector.
+* Hidden columns ``file_modified_time`` and ``file_size`` added to Hive connector.
+* Partition lease support added to Hive connector.
+
+**Details**
+===========
+
+General Changes
+_______________
+* Fix the Aggregation IF to FILTER rule by not unwrapping the IF expression inside the Aggregation. The rule now rewrites ``AGG(IF(condition, expr))`` to ``AGG(IF(condition, expr)) FILTER (WHERE condition)``.
```

Comment:
Maybe we wanna rephrase it more towards user-facing? Something like "Fix the query crash (and explain what crash) with shape ``AGG(IF(condition, expr))`` ..."

Reply:
@yuanzhanhku  can you help with providing more user oriented phrasing that explains what crash was fixed for which query shapes?

Reply:
* Fix the query failures for queries with shape ``AGG(IF(condition, expr))`` where ``expr`` could return exceptions for rows not matching ``condition``. For example, ``SUM(IF(CARDINALITY(array) > 0, array[1])))`` previously could be rewritten to ``SUM(array[1]) FILTER(WHERE CARDINALITY(array) > 0)`` where ``array[1]`` might be evaluated for rows not matching the filter and return out of bound errors.

Reply:
How about
```
* Fix query failures for queries with shape ``AGG(IF(condition, expr))`` where expr could return exceptions for rows not matching ``condition``. These failures occurred when configuration property ``optimizer.aggregation-if-to-filter-rewrite-enabled`` is enabled.
```

Reply:
Looks good. Thanks.